qualification/pytest-jenkins.ini

```diff
@@ -8,6 +8,8 @@ product_name = jenkins_qualification_cbf
 interface = enp193s0np0
 interface_gbps = 150
 use_ibv = true
+# This core ordering distributes load across the CCDs
+cores = 0 4 8 12 1 5 9 13 2 6 10 14 3 7 11 15
```

Comment:
For anyone else coming back to this PR, discussing with Bruce on the ordering and he informs me that
1. It is a 'custom list', but by design (as per the comment), and
2. It's quite specific to our/a 16-core CPU.

This is, of course, for our own internal machines, so it's not like we need to accommodate different CPU core counts (just yet).

Reply:
I've added a comment that it's for an Epyc 7313P.